README.asciidoc

```diff
@@ -316,15 +316,12 @@ The syntax for such an entry is
 
 [source,toml]
 ----
-[[semantic_tokens]]
-token = "variable"
-face = "const_variable_declaration"
-modifiers = ["constant", "declaration"]
+"[token][+modifier1[+modifier2]...]" = "face"
 ----
 
-where `token` is the token's name as reported by the language server (see `lsp-capabilities`), `face` is the face that will be applied in Kakoune (you'll want to define these in your theme/config) and `modifiers` is an array of modifier names (also reported by the language server). `modifiers` may be omitted, but `token` and `face` are required.
+where `token` is the token's name as reported by the language server (see `lsp-capabilities`), `face` is the face that will be applied in Kakoune (you'll want to define these in your theme/config) and `modifiers` is an array of modifier names (also reported by the language server). `modifiers` or `token` may be omitted. An entry with no token will match any token (but should still have modifiers)
```

Comment:
The part about the modifiers array needs to be changed as well, and with more components being optional, examples might help to clarify functionality:

```suggestion
where `token` is the token's name as reported by the language server (see `lsp-capabilities`), `face` is the face that will be applied in Kakoune (you'll want to define these in your theme/config) and each `modifier` is a modifier name (also reported by the language server).

If the `token` or all `modifier`s are omitted, they will act as "match anything" for their respective category.
Example:
- `"variable" = "my_face"` will match any `variable` token, regardless of modifiers
- `"+readonly" = "my_face"` will match any token that's flagged as `readonly`
- `"" = "my_face"` will match everything. You should avoid this, as it will result in huge, inefficient highlighter ranges. Use a generic `fill` highlighter from Kakoune instead.
```